scripts/build/p-libcxx.inc

```diff
@@ -53,7 +59,11 @@ build_libcxx() {
     export LLVM_COMPILER_PATH="$(dirname "${BITCODE_CC}")"
 
     cmake "${cmake_flags[@]}" "${LIBCXX_SRC}/llvm"
-    make cxx -j$(nproc) || make cxx
+   if [[ "${LLVM_VERSION_SHORT}" -ge "14" ]]; then
+     make runtimes - j"$(nproc)" || make cxx || return 1
+   else
+     make cxx - j"$(nproc)" || make cxx || return 1
```

Comment:
This looks like the original intent was to run a parallel make followed by a serial make of the same target. This change introduces two problems:
1. There is a blank between the `-` and the `j` that should not be there
2. it should probably also run `make runtimes` in the fallback position of the then-case